examples/asr/librispeech_conformer_rnnt_biasing/score.sh

```diff
@@ -0,0 +1,2 @@
+dir=experiments/librispeech_clean100_suffix600_tcpgen500_sche30_nodrop/decode_test_clean_b10_KB1000/
```

Comment:
The path can be changed by input argument like `$1`. Could you also add a comment on how to use this script?